### ISSUE TITLE
[Question] Is there any way to change the continuation prompt?

### ISSUE THREAD

**Issue by nasyxx** [jupyter/qtconsole#269]
# Is there any way to change the continuation prompt?

Sometimes, I need to copy multiple lines of code from the generated HTML.

for example:

```
In [1]: (12,
   ...:  12,
   ...:  12)
```

In fact, I only want the result without `...:`:

```
>> (12,
    12,
    12)
```

I understand that this function can be directly modified to achieve the effect I want. But too annoying, is there any easy way to do this?

https://github.com/jupyter/qtconsole/blob/1ee8489b9994b0b12d97a6a1639539926b59517a/qtconsole/jupyter_widget.py#L508

## Version and main components

macOS 10.13.2
Python Version: 3.6.4
qt: stable 5.10.0
pyqt: stable 5.10

**Comment by wmvanvliet** [comment]
I don't think we expose this as a configurable parameter at the moment. Feel free to submit a PR.
